Source/Controllers/MainViewControllers/SPDatabaseDocument.m

```diff
@@ -417,6 +417,23 @@ - (void)setConnection:(SPMySQLConnection *)theConnection
     // Ensure the connection encoding is set to utf8 for database/table name retrieval
     [mySQLConnection setEncoding:@"utf8mb4"];
 
+    // Check if skip-show-database is set to ON
+    SPMySQLResult *result = [mySQLConnection queryString:@"SHOW VARIABLES LIKE 'skip_show_database'"];
+    [result setReturnDataAsStrings:YES];
+    if(![mySQLConnection queryErrored]) {
```

Comment:
Can we combine the ifs into one?

Reply:
It's done. 
`SHOW VARIABLES...` is used more than once to retrieve a single value. I wondered if it was a good idea or not  to create a common method for this. But that was not really the purpose of this change.